src/newgrf_station.cpp

```diff
@@ -503,7 +503,7 @@ uint32 Waypoint::GetNewGRFVariable(const ResolverObject &object, byte variable,
 				cargo += st->goods[cargo_type].cargo.TotalCount();
 			}
 			break;
-
+			
```

Comment:
whitespace change / addition to EOL?

Reply:
I don't know where that's from. It's also a thing above and below that switch/case statement. Might be because of Visual Studio auto formatting files? Don't know.